### DISCUSSION TITLE
What does "Enlightened" mean?

### DISCUSSION THREAD

**Discussion by MrSunshyne** [vercel/satori#218] [General]
The readme starts with describing the project as 

```
Satori: Enlightened library to convert HTML and CSS to SVG.
```

What does "Enlightened" mean in this context?

**Comment by styfle** [comment]
Satori solves a problem that previously required an entire browser implementation (rendering HTML/CSS). 

Therefore, It’s much lighter (aka smaller and faster). 

Also, Satori is often translated as “Enlightenment”. 

- Blog post: https://vercel.com/blog/introducing-vercel-og-image-generation-fast-dynamic-social-card-images
- Related question: https://github.com/vercel/satori/discussions/176#discussioncomment-3887405
- Wikipedia: https://en.m.wikipedia.org/wiki/Satori

**Reply by MrSunshyne** [reply]
Thanks for the reply. 
I was confused because “Enlightenment” usually means to “Shed light onto something” rather than describing the weight of something.

I love the project btw, <3 it addresses an issue I've been trying to solve for years in a very accessible way.

**Reply by styfle** [reply]
Its a [pun](https://www.dictionary.com/browse/pun) - the use of words that are alike or nearly alike in sound but different in meaning; a play on words.
